rm-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/rm/client/TreeView.java

```diff
@@ -46,73 +45,76 @@
 import com.smartgwt.client.widgets.tree.TreeGrid;
 import com.smartgwt.client.widgets.tree.TreeGridField;
 import com.smartgwt.client.widgets.tree.TreeNode;
-import com.smartgwt.client.widgets.tree.events.NodeClickEvent;
-import com.smartgwt.client.widgets.tree.events.NodeClickHandler;
 
 
 /**
  * Displays current nodes in a hierarchical tree view
  * <p>
  * NodeSource > Host > Node
- * 
- * 
- * 
- * 
- * @author mschnoor
  *
+ * @author mschnoor
```

Comment:
Author is not needed here

Reply:
I did not add author. It was already there. I do not dare to remove it.